src/Cronofy/Cronofy.csproj

```diff
@@ -42,6 +42,9 @@
     <Reference Include="System.Runtime.Serialization" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="CronofyApiKeyClient.cs" />

```

Comment:
"API key" could be a number of things. I think this needs to be more explicit, something like `CronofyAdminApiClient` for example.

Reply:
👍